### ISSUE TITLE
[QUESTION] HOW TO UPDATE?

### ISSUE THREAD

**Issue by myfirstevercode** [justinleewells/pogo-optimizer#128]
I already installed with DOCKER but if there's a new update how can i update my old copy? just copy and paste?

**Comment by nubs** [comment]
You can re-pull the docker image and spin up another container:
```bash
docker pull justinleewells/pogo-optimizer
docker run -d -p 3000:3000 -p 8081:8081 -it justinleewells/pogo-optimizer
```

**Comment by johnflavin** [comment]
If you built your own docker image using the source code and the Dockerfile, you'll have to build again. Just re-run `docker build -t pogo .`. If that doesn't work, docker may just be re-using the layers from its cache. You can force it to not do that with `docker build --no-cache -t pogo .`.

**Comment by myfirstevercode** [comment]
thank you

**Comment by petacz** [comment]
Won't there be any issue with certificates? Because I suppose that new certificates will be created, right?

**Comment by nubs** [comment]
With a new docker image, yeah, there will be a new cert.  I should open up a PR to make the cert persist.

**Comment by daveenguyen** [comment]
So I created #163 with possibly a way. The cert can still change. Another way is to update the command we use to run the image to also use **data volumes**. I am not exactly sure yet, but here's a link https://docs.docker.com/engine/tutorials/dockervolumes/

**Comment by nubs** [comment]
Yeah, I was going to add a data volume to things, but your approach works nicely and handles most cases pretty easily.  It also potentially solves some other problems people have been having due to the cert taking a while to generate.

**Comment by justinleewells** [comment]
I'm still kinda partial to the data volumes approach. It could be used to persist player data as well.

Additionally, the cert generation stuff should have been fixed in justinleewells/pogo-optimizer@900c698.
